examples/disaggregated/simpler_example/run_agg.sh

```
@@ -0,0 +1,41 @@
+
+if [ -n "${AGG_PROCESS_ID}" ] || [ -n "${DISAGG_PROCESS_ID}" ]; then
+    kill -9 ${AGG_PROCESS_ID} ${DISAGG_PROCESS_ID}
+fi
+
+echo "Starting aggregate server"
+CUDA_VISIBLE_DEVICES=${CUDA_VISIBLE_DEVICES:-0} trtllm-serve ${MODEL} \
+    --host ${HOST} --port ${PORT} \
+    --config ${CONFIG} 2>&1 | tee log_agg &
+
+echo "Waiting for the aggregate server to start"
+while [ $(curl -s -o /dev/null -w "%{http_code}" "http://${HOST}:${PORT}/health") -ne 200 ]; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)